build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/JdkInfo.cs

```diff
@@ -26,32 +27,30 @@ public class JdkInfo : Task
 
 		public override bool Execute ()
 		{
-			Log.LogMessage (MessageImportance.Low, $"Task {nameof (JdkInfo)}");
-			Log.LogMessage (MessageImportance.Low, $"  {nameof (Output)}: {Output}");
-			Log.LogMessage (MessageImportance.Low, $"  {nameof (AndroidNdkPath)}: {AndroidNdkPath}");
-			Log.LogMessage (MessageImportance.Low, $"  {nameof (AndroidSdkPath)}: {AndroidSdkPath}");
-			Log.LogMessage (MessageImportance.Low, $"  {nameof (JavaSdkPath)}: {JavaSdkPath}");
-
-			var androidSdk    = new AndroidSdkInfo (CreateTaskLogger (this), AndroidSdkPath, AndroidNdkPath, JavaSdkPath);
+			var logger        = CreateTaskLogger (this);
+			var androidSdk    = new AndroidSdkInfo (logger, AndroidSdkPath, AndroidNdkPath, JavaSdkPath);
 			try {
-				var javaSdkPath = androidSdk.JavaSdkPath;
-				if (string.IsNullOrEmpty(javaSdkPath)) {
+				if (string.IsNullOrEmpty(androidSdk.JavaSdkPath)) {
```

Comment:
This conditional is no longer necessary or relevant, because *later* we hit `JdkInfo.GetKnownSystemJdkInfos()`.  This entire conditional can be removed.